### PR TITLE
Create docs_config.js for Elixir & Hex

### LINKED ISSUE #30: Create docs_config.js for Elixir, ExDoc, Hex etc
Ref: https://github.com/hexpm/hexdocs/pull/9

### PR DESCRIPTION
**Pull request by wojtekmach**
Closes #30

### CODE REVIEW

File: priv/scripts/elixir.sh [hunk 1]
```diff
@@ -108,6 +108,13 @@ function upload_docs {
     pushd ${app}
     gsutil -m -h "cache-control: public,max-age=3600" -h "x-goog-meta-surrogate-key: docspage/${app}/${version}" -h "x-goog-meta-surrogate-control: public,max-age=604800" rsync -d -r . "gs://hexdocs.pm/${app}/${version}"
     fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}/${version}"
+
+    mkdir tmp
+    mv docs_config.js tmp/
+    gsutil -m -h "cache-control: public,max-age=3600" -h "x-goog-meta-surrogate-key: docspage/${app}/docs_config.js" -h "x-goog-meta-surrogate-control: public,max-age=604800" rsync tmp "gs://hexdocs.pm/${app}"
```
Comment (ericmj) [inline priv/scripts/elixir.sh]: How come `rsync` instead of `cp`?

Comment (wojtekmach) [reply priv/scripts/elixir.sh]: oh if that's gonna work even better. I was assuming rsync was for that gs:// url, will try this!

Comment (ericmj) [reply priv/scripts/elixir.sh]: I think any command that accepts a local path will also accept a `gs://` URL.

**Comment by ericmj** [comment]
Rebase against master to fix CI errors.

File: priv/scripts/elixir.sh [hunk 2]
```diff
@@ -110,7 +110,7 @@ function upload_docs {
     fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}/${version}"
 
     gsutil -m -h "cache-control: public,max-age=3600" -h "x-goog-meta-surrogate-key: docspage/${app}/docs_config.js" -h "x-goog-meta-surrogate-control: public,max-age=604800" cp docs_config.js "gs://hexdocs.pm/${app}"
-    fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}/docs_config.js"
+    fastly_purge_hexdocs_path "/${app}/docs_config.js"
```
Comment (ericmj) [inline priv/scripts/elixir.sh]: Probably not an issue but we will have double `/` here.

File: priv/scripts/hex_docs.sh [hunk 3]
```diff
@@ -32,7 +32,7 @@ function push {
   fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}/${version}"
 
   gsutil -m -h "cache-control: public,max-age=3600" -h "x-goog-meta-surrogate-key: docspage/${app}/docs_config.js" -h "x-goog-meta-surrogate-control: public,max-age=604800" cp docs_config.jsk "gs://hexdocs.pm/${app}"
-  fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}/docs_config.js"
+  fastly_purge_hexdocs_path "/${app}/docs_config.js"
```
Comment (ericmj) [inline priv/scripts/hex_docs.sh]: Same here.

**Pull request merged**